src/background.ts

```diff
@@ -133,6 +133,8 @@ function generateLocalQuery(message: { context: string, dims: [number, number],
     } as IMAGERequest;
 }
 
+export var requestSent:Boolean
```

Comment:
not used anywhere

Reply:
Yeah, it was for testing. Will remove it!